app/controllers/api/episodes_controller.rb

```diff
@@ -3,4 +3,8 @@ class Api::EpisodesController < Api::BaseController
   represent_with Api::EpisodeRepresenter
   filter_resources_by :podcast_id
   find_method :find_by_guid
+
+  def scoped(relation)
+    relation.published.released
```

Comment:
yeah, and I am thinking ahead to knowing the episode or podcast url before it is public, and my current thought is even to create a stub podcast or episode entry sooner rather than later in the create/save process.

Even without that feeder, has a notion of episodes w/o audio fully processed that are not in the feed yet.